AirportTicketBookingSystem/AirportTicketBookingSystem.Tests/ServicesTests/BaseServiceTests.cs

```diff
@@ -0,0 +1,116 @@
+﻿using AirportTicketBookingSystem.Interfaces;
+using AirportTicketBookingSystem.Services;
+using AirportTicketBookingSystem.Tests.TestDoubles.MyProject.Tests.TestDoubles;
+using FluentAssertions;
+using Moq;
+
+namespace AirportTicketBookingSystem.Tests.ServicesTests
+{
+    public class BaseServiceTests
+    {
+        private readonly Mock<IRepository<TestEntity>> _mockRepo;
+        private readonly BaseService<TestEntity> _service;
+
+        public BaseServiceTests()
+        {
+            _mockRepo = new Mock<IRepository<TestEntity>>();
+            _service = new TestService(_mockRepo.Object);
+        }
+
+        private class TestService : BaseService<TestEntity>
+        {
+            public TestService(IRepository<TestEntity> repo) : base(repo) { }
+        }
+
+        [Fact]
+        public void GetById_Found()
```

Comment:
unit tests should follow this convention : `MethodName_StateUnderTest_ExpectedBehavior`. Lets update it for all tests please.